test/integration/send_slide_to_sign_test.rb

```diff
@@ -17,6 +17,8 @@ class SendSlideToSignTest < Capybara::Rails::TestCase
     sign_out
     sign_in users(:ross)
     assert page.has_content?(/James Slide needs to be approved to .* Ross Sign/)
+    assert page.has_content?(/October 5, 2016/)
+    assert page.has_no_content?(/Ending on/)
```

Comment:
It's not clear to me what's being tested here. This is where second error message argument can be useful:

    assert page.has_no_content?(/Ending on/), "Explain assert or what went wrong here."